voxblox/include/voxblox/mesh/mesh_integrator.h

```diff
@@ -42,19 +42,20 @@
 
 namespace voxblox {
 
+struct MeshIntegratorConfig {
```

Comment:
we moved this outside the mesh integrator, because otherwise this config struct needs to be templated on the VoxelType.